numpy/lib/nanfunctions.py

```diff
@@ -1824,6 +1834,10 @@ def nanstd(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue,
     where : array_like of bool, optional
         Elements to include in the standard deviation.
         See `~numpy.ufunc.reduce` for details.
+    mean : optionally provide the mean to prevent recalculation, the mean
+          should have a shape as if it was calculated with keepdims = True
+          and the axis the same as used in the call to this var function.
+          For masked arrays this argument is ignored.
 
         .. versionadded:: 1.22.0
```

Comment:
This needs a bit of tweaking like the other ones. As well, the 1.22.0 version added line belongs to `where`.